docs/mom/database.md

```diff
@@ -0,0 +1,34 @@
+# [Kingdom Hearts Melody of Memory](index.md) Database
+
+Most of the game logic is able to be manipulated using a SQLite database located in: <br>
+`StreamingAssets\SQLite\Table.db`
+
+## Documented Tables
+| Table Name | Notes |
+|--|--|
+| GameSceneFlowTable | Defines menu layout, eg. Title>Tutorial>Main Screen, comments describe what each flow does, translation required however |
+| MusicTable | Defines music track properties, like difficulty values, item drops and drop rates |
+<br>
+
+## Noteworthy Details
+* 3 regions are defined in MusicTable, "Japan", "World", and "Asia", with boolean values on each song defining if they are "included" in that region.
+    * This may be used for different translations of music like Hikari/Simple and Clean
+
+## Example Edits
+Using a SQLite DB editor, such as "DB Browser for SQLite" execute these SQL queries:
+
+**Disabling first start tutorial** <br>
```

Comment:
Use the `### Disabling first start tutorial` to mark a subcategory inside a category of `##` rather than using **bold text**. That will help us to style the text in the future. Also never use `<br>` in markdown code :)